index.md

```diff
@@ -1,6 +1,6 @@
 # Project reference setup with ansible & vagrant
 
-[![Build Status](https://travis-ci.org/wunderkraut/WunderMachina.svg?branch=centos7)](https://travis-ci.org/wunderkraut/WunderMachina)
+[![Build Status](https://travis-ci.org/wunderkraut/WunderTools)](https://travis-ci.org/wunderkraut/WunderTools)
```

Comment:
Still broken, in another way :)

![image](https://cloud.githubusercontent.com/assets/1324225/23759312/1657adfc-04f5-11e7-9c8a-9c5aba116f06.png)

The first link should be pointing to an image filename.

Try this:
```md
[![Build Status](https://travis-ci.org/wunderkraut/WunderTools.svg?branch=master)](https://travis-ci.org/wunderkraut/WunderTools)
```

(Click the badge at https://travis-ci.org/wunderkraut/WunderTools, select Markdown, copy.)